Script-2(final).sql

```diff
@@ -74,47 +74,60 @@ CREATE TABLE warehouse(
 	term_of_use varchar(255),
 	banner_url varchar(255),
 	thumbnail_url varchar(255),
-	discount_type varchar(20),
+	discount_type bigint,

```

Comment:
phụng thêm chỗ này not null nha, với lại thêm chữ id tại cái này là khóa ngoại của bảng discount type